src/SfdxProjectBuilder.groovy

```diff
@@ -963,6 +989,60 @@ class SfdxProjectBuilder implements Serializable {
     }
   }
 
+    
+   // Runs PMD analysis on code
+   // Checks if PMD is installed before running
+  private void runPmdAnalysis() {
```

Comment:
@FernandoQuezado -- At which stage would the `runPmdAnalysis()` be executed?  I didn't find any other mention of it above in the various stages.